testsuites/listener/shared/client_sg/test_replication.py

```diff
@@ -139,6 +141,106 @@ def test_initial_pull_replication(setup_client_syncgateway_test, continuous):
 @pytest.mark.listener
 @pytest.mark.syncgateway
 @pytest.mark.replication
+@pytest.mark.session
+@pytest.mark.parametrize("num_docs, need_attachments, replication_after_backgroundApp", [
+    (1000, True, False),
+    (10000, False, False),
+    (10000, False, False),
+    (100000, False, False),
+    (100000, False, False),
+    (100000, False, True)
+])
+def test_initial_pull_replication_background_apprun(setup_client_syncgateway_test, num_docs, need_attachments, replication_after_backgroundApp):
```

Comment:
Maintain the length to less than 80 col. You can indent arguments one below another